configs/config.json.example

```diff
@@ -32,6 +32,10 @@
     "longer_eggs_first": true,
     "evolve_captured": false,
     "release_pokemon": true,
+    "spin_forts": {
```

Comment:
There is already key `spin_forts`, don't they conflict? Shouldn't `avoid_circles` and `max_circle_size` be on the same level as `spin_forts` ? (Sorry if I'm wrong, not a Python man)